_posts/en/newsletters/2021-01-13-newsletter.md

```diff
@@ -0,0 +1,223 @@
+      simultaneously better than honest peers at all of these characteristics
+      to partition a node."
+  a2link="https://bitcoincore.reviews/20477#l-83"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 0.21.0rc5][Bitcoin Core 0.21.0] is a Release Candidate (RC)
+  for the next major version of this full node implementation and its
+  associated wallet and other software.  Jarol Rodriguez has written an
+  [RC testing guide][] that describes the major changes in the release
+  and suggests how you can help test them.
+
+- [LND 0.12.0-beta.rc5][LND 0.12.0-beta] is the latest release candidate
+  for the next major version of this LN node.  It makes [anchor
+  outputs][topic anchor outputs] the default for commitment transactions
+  and adds support for them in its [watchtower][topic watchtowers]
+  implementation, reducing costs and increasing safety.  The release
+  also adds generic support for creating and signing [PSBTs][topic psbt]
+  and includes several bug fixes.
+
+{% comment %}<!--
+- Bitcoin Core 0.20.2rc1 and 0.19.2rc1 are expected to be
+  [available][bitcoincore.org/bin] sometime after the publication of
+  this newsletter.  They contain several bug fixes, such as an
+  improvement described in [Newsletter #110][news110 bcc19620] that will
+  prevent them from redownloading future taproot transactions that they
+  don't understand.
+-->{% endcomment %}
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #18077][] introduces support for a common automatic port
+  forwarding protocol, [NAT-PMP][rfc 6886] (Network Address Translation Port
+  Mapping Protocol).  A listening Bitcoin client started with the `-natpmp`
+  configuration parameter will automatically open the listening port on NAT-PMP-enabled routers.
+  NAT-PMP support is added in parallel to the existing UPnP (Universal Plug and Play) support
+  which had been disabled by default in Bitcoin Core 0.11.1 after multiple
+  security issues. In contrast to UPnP, NAT-PMP uses fixed-size UDP packets
+  instead of XML parsing and is therefore considered [less risky][laanwj
+  natpmp]. This change also transitively adds support for [PCP][rfc 6887] (Port
+  Control Protocol), the backward-compatible successor to NAT-PMP.
+
+- [Bitcoin Core #19055][] adds the MuHash algorithm so that a future implementation
+  can use it.  As covered in
```

Comment:
"so that a future implementation can use it" is a bit tautological. I think the rest of the write-up explains well why and how this can be used, so how about:

`- [Bitcoin Core #19055][] adds a C++ implementation of the [MuHash algorithm](https://cseweb.ucsd.edu/~mihir/papers/inchash.pdf).`

as the first sentence?

Reply:
@jnewbery hopefully better now?  It's still a bit of a tautology, but I do want to make it clear that there's no way for anyone to use the feature yet (excluding tests).